### PR TITLE
Use govuk buildproject in jenkinsfile

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/BQCjFGpq/3-investigate-travel-advice-publisher-using-buildproject-groovy-script

This PR simplifies Travel Advice's Jenkinsfile to use `govuk.buildProject`.
In order to do this some dependencies were updated to satisfy the `gov-lint-sass` build step.
I've tested the sass changes on integration and can't see any difference with staging.

### CODE REVIEW

File: app/assets/stylesheets/scaffold.scss [hunk 1]
```diff
@@ -1,17 +1,18 @@
...
-// fixed navbar
+$medium-dark-grey: #555;
+
+// Increase padding on footer to account for fixed navbar
 .page-footer {
   margin-bottom: 63px;
 }
 
 // Override the font-size for main textareas
-.formtastic textarea.form-control {
```
Comment (fofr) [inline app/assets/stylesheets/scaffold.scss]: Not sure if this change will have unexpected side effects. eg .form-control could be on input elements too. Could disable the sass rule on this selector for now?

File: app/assets/stylesheets/scaffold.scss [hunk 2]
```diff
@@ -1,17 +1,18 @@
...
+.formtastic .form-control {
   font-size: 17px;
   line-height: 26px;
-  color: #555;
+  color: $medium-dark-grey;
 }
 
-fieldset.inputs {
+.inputs {
```
Comment (fofr) [inline app/assets/stylesheets/scaffold.scss]: Wonder if changing the class to `fieldset-inputs` would be worthwhile. `.inputs` seems to generic and could clash.

File: app/assets/stylesheets/sortable_accordion.scss [hunk 3]
```diff
@@ -1,3 +1,5 @@
+$light-yellow: #ff9;
```
Comment (fofr) [inline app/assets/stylesheets/sortable_accordion.scss]: I enjoyed misreading this as `slight-yellow` 😄

**Comment by steventux** [comment]
Closing briefly while I look at dependency downgrade for `sass`

**Pull request closed without merging**